user_guide_src/source/incoming/controllers.rst

```diff
@@ -213,16 +232,16 @@ Private methods
 ===============
 
 In some cases, you may want certain methods hidden from public access.
-In order to achieve this, simply declare the method as being private
-or protected and it will not be served via a URL request. For example,
-if you were to have a method like this::
+To achieve this, simply declare the method as private or protected. 
+That will prevent it from being served by a URL request. For example,
+if you were to define a method like this::
 
 	protected function utility()
 	{
 		// some code
 	}
 
-Trying to access it via the URL, like this, will not work::
+then trying to access it using the following URL will not work::
 
 	example.com/index.php/blog/utility/
```

Comment:
Maybe make this match the new example controller?
`example.com/index.php/helloworld/utility/`

Reply:
Good idea. Done.

Reply:
Thanks! Looks great, merging.